core/common/modules/connect/module.php

```diff
@@ -203,17 +203,4 @@ public function get_subscription_plans( $context ) {
 			],
 		];
 	}
-
-	protected function get_init_settings() {
```

Comment:
Restore, Why it's deleted?

Reply:
Not sure why to restore it. it is a code that I added, but then we decided  to change the settings only for the Admin Top Bar, not widely

Reply:
The method `get_init_settings` existed before your changes.

Reply:
The logic inside the get_init_settings is related to the admin top bar, after the changes in this PR and the PR in Pro it is not needed anymore.